anvio/__init__.py

```diff
@@ -3669,7 +3669,27 @@ def TABULATE(table, header, numalign="right", max_width=0):
                      "a comma-separated list. The default stats are 'detection' and "
                      "'mean_coverage_Q2Q3'. To see a list of available stats, use this flag "
                      "and provide an absolutely ridiculous string after it (we suggest 'cattywampus', but you do you)."}
-    )
+    ),
+    'mode': (
+        ['--mode', '-M'],
+        {'default': None,
+         'metavar': 'structure',
+         'type': str,
+         'help': 'Use this flag to set mode to structure or sequence.'}
+    ),
+   'prostt5-weight-dir': (
```

Comment:
Why not call it `prostt5-data-dir`?

Indentations of new entries in `__init__` do not much each other.

Reply:
please don't forget to change the help files with the new parameter name too.

Reply:
Frankly it seems more correct to use _prostt5-weight-dir_ here. Because what we downloaded is a model's weight file and Foldseek uses it with the same name, for example:` foldseek createdb db.fasta db --prostt5-model weights`

Reply:
It doesn't matter though. As far as our users are concerned, it is yet another data they are downloading. Regardless what individual data items mean to individual processes, we can have a common vocabulary we bring in from the outside world. What we download from COGs is FASTA files. What we download from KEGG is models. We call all of them data on anvi'o side.